pkg/trace/agent/agent.go

```diff
@@ -439,3 +439,8 @@ func newEventProcessor(conf *config.AgentConfig) *event.Processor {
 
 	return event.NewProcessor(extractors, conf.MaxEPS)
 }
+
+// SetGlobalTags sets global tags to the current configuration
```

Comment:
Does this really set global tags to the current configuration? It looks like it sets them to whatever is passed into this function.

Reply:
✅